README.md

```diff
@@ -60,11 +60,21 @@ blog = build_blueprint(
         exclude_tags=[26, 34],
         per_page=12,
         feed_description="The Ubuntu Blog Feed",
-        api=Wordpress(session=session),
+        api=BlogAPI(session=session),
     )
 )
 ```
 
+If you wish to get the raw format of the articles use the `Wordpress` class instead of `BlogAPI`:
```

Comment:
I dunno if you need to mention this. I think there's only 1 recommended way to set up this module and that's with the `BlogAPI`. The `Wordpress` class is just for purity - in that we could extract that class for use somewhere else if we wanted.

I don't feel strongly, but this seems to complicate the README for little utility.